run-reader.sh

```diff
@@ -1,31 +1,39 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
-export JAVA_OPTS="-Xdebug -Xrunjdwp:transport=dt_socket,address=8002,server=y,suspend=n"
+export JAVA_OPTS=" -Xmx1204m -Xdebug -Xrunjdwp:transport=dt_socket,address=8002,server=y,suspend=n"
 export PORT="${PORT:-8082}"
 
 # Restolino configuration
+export RESTOLINO_STATIC="src/main/resources/files"
 export RESTOLINO_CLASSES="zebedee-reader/target/classes"
-export PACKAGE_PREFIX=com.github.onsdigital.zebedee.reader.api
-export CONTENT_DIR="content"
+export PACKAGE_PREFIX=com.github.onsdigital.zebedee
```

Comment:
Is there any reason why this has been changed from the original value of  com.github.onsdigital.zebedee.reader.api?

Reply:
I didn't consciously make these changes - I was just trying to puzzle out what'd happened - I think I 'created' run-reader.sh by copying run.sh without realising it was already there, so these are values that are set in run.sh. I'll revert it and clean up